src/ipc-handlers.ts

```diff
@@ -405,6 +406,19 @@ export async function startServer(
 		console.log(
 			`Domain ${ server.details.customDomain } added to hosts file for port ${ server.details.port }`
 		);
+
+		try {
+			const proxyStarted = await startProxyServer();
+			if ( ! proxyStarted ) {
+				console.warn(
+					'Failed to start custom domain proxy server - custom domains will require port numbers'
+				);
+			}
+			console.log( 'Custom domain proxy server started successfully' );
```

Comment:
I noticed we display `Custom domain proxy server started successfully` each time we start a server with custom domain.
I think it's innocuous as we also display the `Domain example.wp.local added to hosts file for port 8900`

Reply:
I removed redundant logging in https://github.com/Automattic/studio/pull/1089/commits/67bdbcfe656cb99949fd15cf9485a9446b7f0e5e and moved the hosts log message to the proper place in https://github.com/Automattic/studio/pull/1089/commits/f29028e26cc859fb28661ade2602a934030b1498